client/src/client/__init__.py

```diff
@@ -104,11 +104,18 @@ def __init__(self) -> None:
 
         # handle consecutive startups
         else:
-            # TODO: this while loop
             feedback = ""
             retries = 3
+            password_needed = False
 
-            while True:
+            try:
+                self.password = self.startup.get_key()
+                self._load_data(password=self.password)
+            except Exception as e:
```

Comment:
Did linter let you have bare exception here?

Reply:
Yes, I had no problems with linters here. A more thorough error check is done for when attempting decryption with user's passphrase so I thought that using Exception here is the best choice of action.